uint32 number
uint32[] song
---
bool success
